__tests__/app.test.js

```diff
@@ -0,0 +1,64 @@
+const request = require('supertest')
+const app = require('../app')
+
+const db = require('../db/connection')
+const seed= require('../db/seeds/seed')
+
+const data = require('../db/data/test-data')
+
+beforeEach(() => {
+    
+    return seed(data)
+})
+
+afterAll(() => {
+     return db.end();
+})
+
+describe('app', () => {
+   it('status: 404 for invalid path', () => {
+       return request(app)
+           .get('/invalid path')
+           .expect(404)
+           .then(({body : { msg}}) => {
+           expect(msg).toBe('Not Found')
+               
+       })
+   }); 
+    
+    describe('/api', () => {
+        
+        describe('GET/api/topics', () => {
```

Comment:
This is a really good test, checking the object that is returned has the correct length and looks exactly how it should do.